tests/test_C0_data_processing.py

```diff
@@ -51,6 +51,7 @@
     BUS_SUFFIX,
```

Comment:
Maybe add complementary messages to the `assert` calls which help the future developpers?

Reply:
Added in c7c72db2c7a0be0f21d5337d794dea9b098113e2 and d91ed8b23db4eff7bd50341415ff6407d1a7ca68.